cmd/evm/README.md

```diff
@@ -45,27 +54,175 @@ Command line params that has to be supported are
 
 ```
 
-### Error codes and output
+#### Objects
+
+The transition tool uses JSON objects to read and write data related to the transition operation. The
+following object definitions are required.
+
+##### `alloc`
+
+The `alloc` object defines the prestate that transition will begin with.
+
+```go=
+// Map of address to account definition.
+type Alloc map[common.Address]Account
+
+// Genesis account. Each field is optional.
+type Account struct {
+    Code       []byte                           `json:"code"`
+    Storage    map[common.Hash]common.Hash      `json:"storage"`
+    Balance    *big.Int                         `json:"balance"`
+    Nonce      uint64                           `json:"nonce"`
+    SecretKey  []byte                            `json:"secretKey"`
+}
```

Comment:
For those of us that don't speak go as fluently, how do these get translated into JSON? Especially the `big.Int` fields.

Reply:
I added a note on encoding at the bottom, but basically we use a intermediary type `math.HexOrDecimalN` which can decode json strings that are either hex format or decimal format.